app/views/quotes/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<% @quotes.each do |quote| %>
+  <div>
```

Comment:
Upon further thinking, you're probably better off including `<blockquote>` in the partial itself.